src/containers/Search.jsx

```diff
@@ -44,9 +44,24 @@ class Search extends Component {
       });
   };
 
+  insertionsort = (inputArr) => {
+    let length = inputArr.length;
+    for (let i = 1; i < length; i++) {
+        let key = inputArr[i]
+        let dist = this.generateDistance(key.geometry.location,this.state.lat,this.state.lng,"K");
```

Comment:
this should be the key function;
It would be much better if for each `inputArr[i]` you already map `this.generateDistance` output so that you can use that to get the key value inside the function key parameter i mentioned above;